hathor/transaction/storage/rocksdb_storage.py

```diff
@@ -151,8 +151,16 @@ def get_all_transactions(self) -> Iterator['BaseTransaction']:
             yield tx
 
     def get_count_tx_blocks(self) -> int:
```

Comment:
You might add a parameter here `use_rocksdb_estimation: bool = False`. And I feel it is ok to raise `NotImplementedError` if `use_rocksdb_estimation` is false and it has no indexes.

Another way would be to move this estimation method to new method `get_estimated_count_tx_blocks(...)`. What do you think?

Reply:
Thanks!